Book-A-Desk.Domain/Reservation.Instruction.fs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Book_A_Desk.Domain.Reservation.Commands.Instructions
+
+open Book_A_Desk.Domain.Events
+open Book_A_Desk.Domain.Office.Domain
+open Book_A_Desk.Domain.Reservation
+open Book_A_Desk.Domain.Reservation.Domain
+
+module ReservationInstructions =
```

Comment:
This part is the least interesting bit, but here's a high level overview of what it does. Essentially we build what is called a DSL or Domain Specific Language. Essentially, we add words to our language that describes what an interpreter should do when encountering the command.

So, if an interpreter encounters "AppendEvents", it should "AppendEvents", whatever that means for the interpreter. It could mean to append the events in an in-memory store, or to some cloud database. The point is, the domain does not care what happens, just that it thinks someone should append events.